tests/solidity/test_close_short.py

```diff
@@ -1,11 +1,13 @@
 """Close short market trade tests that match those being executed in the solidity repo"""
 import unittest
+from decimal import Decimal
 
 import elfpy.agents.agent as agent
 import elfpy.markets.hyperdrive.hyperdrive_market as hyperdrive_market
 import elfpy.pricing_models.hyperdrive as hyperdrive_pm
 import elfpy.time as time
 import elfpy.types as types
+from elfpy.pricing_models import yieldspace
```

Comment:
This syntax can lead to circular imports. It works by importing the entire `pricing_models` directory, and then deleting everything besides `yieldspace`. 

I would follow the hyperdrive syntax above: `import elfpy.pricing_models.yieldspace as yieldspace_pm`

Down the road (after we finish the big refactor) I'd like to switch back to the `from` syntax, but for now we should keep everything like this.